prow/github/client.go

```diff
@@ -2722,7 +2724,9 @@ func (c *Client) GetColumnProjectCard(columnID int, cardNumber int) (*ProjectCar
 		return nil, err
 	}
 	for _, card := range cards {
-		if card.ContentID == cardNumber {
+		properContent := strings.Replace(card.ContentURL, "repos/", "", 1)
```

Comment:
Maybe let's break this out into a helper and unit-test it so it's clear what is going on here?

Reply:
There is subtle difference  when you see the link in the card it does not standardize the same way.
Ex: Link to issue in the card is: 
"content_url": "https://api.github.com/repos/ii/apisnoop/issues/14"
Where as the actual issue is: 
https://github.com/ii/apisnoop/issues/14

Reply:
If we are parsing the URL perhaps we should parse it first, then operate on the path? Can I have a repo named `repos`?

Reply:
I am only trying to use this logic when the URL string don't match, when a html_url is supplied.